tools/execsnoop.py

```diff
@@ -221,6 +227,10 @@ def parse_uid(user):
 
 bpf_text = bpf_text.replace("MAXARG", args.max_args)
 
+if args.file:
+    file = open(args.file, 'w')
```

Comment:
I wonder if we should open the file in "append" mode or not. I am not sure what would be the expected behaviour.

Reply:
Good question. I think truncating the file is ok, the tools that activates the gadget could be sure that the contents of such file are processed before calling it. But I don't have a strong opinion neither.

Reply:
We can keep it like this and change this later.